### PR TITLE
chore: remove detekt ClassNaming issues

### PR DESCRIPTION
**Pull request by joc-a**
This PR removes `detekt` `ClassNaming` issues and reformats the affected files.

### CODE REVIEW

File: exposed-core/src/main/kotlin/org/jetbrains/exposed/sql/Op.kt [hunk 1]
```diff
@@ -552,7 +559,7 @@ class RegexpOp<T : String?>(
 /**
  * Represents an SQL operator that checks if [query] returns at least one row.
  */
-class exists(
+class Exists(
```
Comment (joc-a) [inline exposed-core/src/main/kotlin/org/jetbrains/exposed/sql/Op.kt:562]: Potential breaking change introduced here. If we don't want this, I can revert this change and suppress the `detekt` warning.

File: exposed-core/src/main/kotlin/org/jetbrains/exposed/sql/Op.kt [hunk 2]
```diff
@@ -563,10 +570,13 @@ class exists(
...
 }
 
+/** Returns an SQL operator that checks if [query] returns at least one row. */
+fun exists(query: AbstractQuery<*>) = Exists(query)
+
 /**
  * Represents an SQL operator that checks if [query] doesn't returns any row.
  */
-class notExists(
+class NotExists(
```
Comment (joc-a) [inline exposed-core/src/main/kotlin/org/jetbrains/exposed/sql/Op.kt:579]: Potential breaking change introduced here. If we don't want this, I can revert this change and suppress the `detekt` warning.

File: exposed-core/src/main/kotlin/org/jetbrains/exposed/sql/Op.kt [hunk 3]
```diff
@@ -100,10 +105,11 @@ infix fun Expression<Boolean>.and(op: Expression<Boolean>): Op<Boolean> = when {
...
     op is AndOp -> AndOp(
-        ArrayList<Expression<Boolean>>(op.expressions.size + 1).also {
-            it.add(this)
-            it.addAll(op.expressions)
-        }
+        ArrayList<Expression<Boolean>>(op.expressions.size + 1)
+            .also {
+                it.add(this)
+                it.addAll(op.expressions)
+            }
```
Comment (bog-walk) [inline exposed-core/src/main/kotlin/org/jetbrains/exposed/sql/Op.kt:112]: Which detekt rule is this based on (and the change above)? Is it something set in the project .yml or based on default detekt settings?

Comment (joc-a) [reply exposed-core/src/main/kotlin/org/jetbrains/exposed/sql/Op.kt:112]: This came from Reformat Code (⌥⌘L), but I don't think it's necessary actually, so I'll remove it.

Comment (joc-a) [reply exposed-core/src/main/kotlin/org/jetbrains/exposed/sql/Op.kt]: Correction: This did not come from Reformat Code. I must have done that before or after Reformat Code and forgot about it.

**Review by e5l — APPROVED** [review]

**Pull request merged**